layer1/executor/tasks/tests/snapshot_test.go

```diff
@@ -0,0 +1,450 @@
+//go:build integration
+
+package tests
+
+import (
+	"context"
+	"fmt"
+	"github.com/alicenet/alicenet/consensus/objs"
+	"github.com/alicenet/alicenet/crypto"
+	"github.com/alicenet/alicenet/crypto/bn256"
+	dkgState "github.com/alicenet/alicenet/layer1/executor/tasks/dkg/state"
+	"github.com/alicenet/alicenet/layer1/executor/tasks/snapshots"
+	"github.com/alicenet/alicenet/layer1/executor/tasks/snapshots/state"
+	"github.com/alicenet/alicenet/layer1/tests"
+	"github.com/alicenet/alicenet/layer1/transaction"
+	"github.com/alicenet/alicenet/logging"
+	"github.com/alicenet/alicenet/test/mocks"
+	"github.com/alicenet/alicenet/utils"
+	"github.com/ethereum/go-ethereum/common"
+	ethCrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"math/big"
+	"math/rand"
+	"testing"
+	"time"
+)
+
+func Test_SnapshotTask(t *testing.T) {
+	n := 5
```

Comment:
`n` should have a more descriptive name.

Reply:
done